app/views/layouts/application.html.erb

```diff
@@ -62,5 +60,16 @@
       </div>
     </div>
   </footer>
+  <!-- Dropdown Structure -->
+  <ul id='dropdown1' class='dropdown-content'>
```

Comment:
This is really good. My only suggestion would be that this id is not really descriptive. Could we use `account-dropdown` or something? What do you think?